### DISCUSSION TITLE
alternate screenshot filename for subtitles

### DISCUSSION THREAD

**Discussion by caramel253** [mpv-player/mpv#15084] [Q&A] [answered]
hello, i would like to know if a very specific request of mine is possible.
i want to, for example, separate the screenshot name for the s key (shot with subtitles) and the S key (shot without subtitles). it would go like this: 
-s key: mpv-shot0001-subtitles.png;
-S key: mpv-shot0001.png;
would that be possible with a command line from the mpv.conf file?
for another example, this is my current formatting for screenshots with subtitles: 
-screenshot-dir="C:\Users\user\Pictures\mpv"
-screenshot-template="mpv-shot%n_subtitles"
in the case for the screenshot with subtitles, the formatting would be like this:
-screenshot-dir="C:\Users\user\Pictures\mpv"
-screenshot-template="mpv-shot%n"

**Comment by llyyr** [comment, marked as answer]
Rebind the keys in your input.conf

`s set screenshot-template "mpv-shot%n-subtitles" ; screenshot`

**Comment by guidocella** [comment]
https://mpv.io/manual/master/#command-interface-screenshot-to-file

**Comment by caramel253** [comment]
i'm sorry, i'm not very proficient with this, but neither did seem to work. the post linked below helped with binding the "s" key for a specific filename, but it doesn't seem to support screenshot templates like "%f" or "%P" for formatting, and i can't set a specific directory either. thank you either way, i don't know if a specific like i mentioned is possible, but i will add more information to my post for clarification in the case it is.

**Reply by llyyr** [reply]
Sorry, I edited the keybind. I didn't type the full command somehow. But you should use screenshot-to-file instead

**Reply by caramel253** [reply]
that worked perfectly now!! thank you very much
